custom_components/opnsense/coordinator.py

```
@@ -51,58 +52,44 @@
-    @_log_timing
-    async def _get_carp_status(self):
-        return await self._client.get_carp_status()
-
-    @_log_timing
-    async def _get_dhcp_leases(self):
-        return await self._client.get_dhcp_leases()
-
-    @_log_timing
-    async def _get_notices(self):
-        return await self._client.get_notices()
+    async def _get_states(self, categories: list) -> Mapping[str, int]:
```

Ruff

Missing type annotation for `self` in method

(ANN101)